eq-author-api/schema/resolvers/base.js

```diff
@@ -27,9 +23,22 @@ const {
   getValidationEntity,
 } = require("../../src/businessLogic/createValidation");
 const uuid = require("uuid");
-const deepMap = require("deep-map");
 const { currentVersion } = require("../../migrations");
 
+const {
+  getSection,
+  getPage,
+  getAnswers,
+  getAnswer,
+  getOption,
+  getValidation,
+  getAvailableMetadataForValidation,
```

Comment:
A few of these don't sound re-used. Like `getAvailableMetadataForValidation`, `getAvailablePreviousAnswersForValidation` and `getQuestionnaireList`.

Can we only put reused things in the utils.